Documentation/cmdref/cilium-operator-aws.md

```diff
@@ -28,7 +28,6 @@ cilium-operator-aws [flags]
       --cnp-status-update-interval duration       Interval between CNP status updates sent to the k8s-apiserver per-CNP (default 1s)
       --config string                             Configuration file (default "$HOME/ciliumd.yaml")
       --config-dir string                         Configuration directory that contains a file for each option
-      --crd-wait-timeout duration                 Operator will exit if CRDs are not available within this duration upon startup (default 5m0s)
```

Comment:
Do we need to deprecate these or can we just remove them for 1.9? If we can remove, then we'll need to update the docs too. @aanm